coldfront/core/project/tests/test_project_removal_request_views.py

```diff
@@ -0,0 +1,611 @@
+from django.test import TestCase
```

Comment:
Can you split these into one class per view and move this file under `tests.test_views`? The overlapping `setUp` logic can be extracted into a base class that they each inherit from.